playwright.config.ts

```diff
@@ -68,51 +68,118 @@ const config: PlaywrightTestConfig<TestOptions> = {
       testMatch: "login.spec.ts",
     },
     {
-      name: "chromium:lxd-5.0-edge",
+      name: "enable-clustering-chrome",
+      // enable clustering for subsequent tests
+      use: { ...devices["Desktop Firefox"], trace: "off" },
```

Comment:
```suggestion
      use: { ...devices["Desktop Chrome"] },
```
This is chrome, and we only disable tracing for login to not leak the password.

Reply:
Do we need a Firefox version as well?